src/ripple/protocol/impl/RippleAddress.cpp

```diff
@@ -59,23 +59,21 @@ static uint128 PassPhraseToKey (std::string const& passPhrase)
 
 static Blob getPublicKey (openssl::ec_key const& key)
 {
-	Blob result (33);
-	
-	key.get_public_key (&result[0]);
-	
-	return result;
+    Blob result (33);
```

Comment:
Given these alternatives, I think the constant is clearest to me.
Blob result(33);
Blob result (33); // public key
Blob result (publicKeyBlobLength);

But if the rest of the system does this I'm not trying to make this code stick out either. So I guess do what the rest of the system does and if we refactor it we'll do them all at the same time.